client/extensions/woocommerce/state/sites/settings/email/reducer.js

```diff
@@ -22,40 +22,53 @@ import {
 	WOOCOMMERCE_EMAIL_SETTINGS_SUBMIT_FAILURE,
 	WOOCOMMERCE_EMAIL_SETTINGS_INVALID_VALUE,
 } from 'woocommerce/state/action-types';
-export default createReducer( null, {
-	[ WOOCOMMERCE_EMAIL_SETTINGS_REQUEST ]: () => {
-		return LOADING;
-	},
 
-	[ WOOCOMMERCE_EMAIL_SETTINGS_REQUEST_SUCCESS ]: ( state, { data } ) => {
-		const options = {};
-		const fromAddress = filter( data, {
-			group_id: 'email',
-			id: 'woocommerce_email_from_address',
-		} );
-		const defaultEmail = isEmpty( fromAddress ) ? '' : fromAddress[ 0 ].default;
-		data.forEach( function( option ) {
-			const def = option.id === 'recipient' ? option.default || defaultEmail : option.default;
-			setWith(
-				options,
-				[ option.group_id, option.id ],
-				{
-					value: option.value,
-					default: def,
-				},
-				Object
-			);
-		} );
+const process_data = data => {
+	const options = {};
+	const fromAddress = filter( data, {
+		group_id: 'email',
+		id: 'woocommerce_email_from_address',
+	} );
+	const defaultEmail = isEmpty( fromAddress ) ? '' : fromAddress[ 0 ].default;
+	data.forEach( function( option ) {
+		setWith(
+			options,
+			[ option.group_id, option.id ],
+			{
+				value: option.value,
+				default: option.default,
+			},
+			Object
+		);
+	} );
 
-		// Decode: &, <, > entities.
-		const from_name = get( options, [ 'email', 'woocommerce_email_from_name', 'value' ], false );
-		if ( from_name ) {
-			options.email.woocommerce_email_from_name.value = decodeEntities( from_name );
+	forEach( [ 'email_new_order', 'email_cancelled_order', 'email_failed_order' ], key => {
+		if ( get( options, [ key, 'enabled', 'value' ] ) !== 'yes' ) {
+			return;
 		}
+		const _default = get( options, [ key, 'recipient', 'default' ] ) || defaultEmail;
+		options[ key ].recipient = {
+			default: _default,
+			value: get( options, [ key, 'recipient', 'value' ] ) || _default,
+		};
+	} );
+
+	// Decode1: &, <, > entities.
```

Comment:
I wrote a new issue for this - it doesn't pick up apostrophes - #21837

Reply:
Feel free to fix here or separately

Reply:
(It happens on master too, fwiw)

Reply:
It is not listed but this branch already handles apostrophes
 
![image](https://user-images.githubusercontent.com/17271089/35435431-01af0abe-028b-11e8-8a35-a6dc8e2ef99b.png)

Reply:
Marking #217837 as fixed by this PR in PRs description.